src/Repository/JobsRepository.php

```diff
@@ -20,23 +20,129 @@ public function __construct(ManagerRegistry $registry)
     {
         parent::__construct($registry, Jobs::class);
     }
-
-    public function save(Jobs $entity, bool $flush = false): void
+    
+    /**
+     * add to db
+     *
+     * @param  mixed $entity
+     * @return bool
+     */
+    public function save(Jobs $entity): bool
     {
         $this->getEntityManager()->persist($entity);
+        $this->getEntityManager()->flush();
 
-        if ($flush) {
-            $this->getEntityManager()->flush();
-        }
+        return $entity->getId() > 0;
     }
-
-    public function remove(Jobs $entity, bool $flush = false): void
+    
+    /**
+     * delete from db
+     *
+     * @param  mixed $entity
+     * @return void
+     */
+    public function remove(Jobs $entity): void
     {
         $this->getEntityManager()->remove($entity);
+        $this->getEntityManager()->flush();
+    }
+    
+    /**
+     * querry builder to update
+     *
+     * @param  mixed $id
+     * @param  mixed $params
+     * @return int
+     */
+    public function update(int $id, array $params): int
+    {
+        $queryBuilder = $this->createQueryBuilder('j');
+
```

Comment:
job name and description are not mandatory fields. Add them to the query only if they exist in $params